docs/work-with-data-apis.md

```diff
@@ -0,0 +1,66 @@
+# Work with unsupported REST Data APIs
+
+HERE Data SDK for C++ supports only a limited number of REST Data APIs. You can find a list of them on our page with the [overall architecture](OverallArchitecture.md).
+
+If you need to work with the REST Data APIs that are currently not supported, you can configure the Data SDK with a unique base URL.
+
+The example below demonstrates how to use the Data SDK to call the API for downloading the difference between two versions of a versioned catalog.
+
+**Example. Download the difference between two versions of a catalog.**
+
+1. Create the `OlpClientSettings` object.
+
+   For instructions, see [Create platform client settings](create-platform-client-settings.md).
+
+2. Get a base URL from the lookup API service.
+
+    For instructions, see the [API Lookup API Developer's Guide](https://developer.here.com/documentation/api-lookup/dev_guide/index.html).
+
+    You can now use this URL as the base for all requests to the REST Data API.
+
+3. Create an `OlpClient` instance with the client settings from step 1 and base URL from step 2.
+
```

Comment:
Add a note maybe and say that this step can be omitted when using `olp::client::ApiLookupClient` instead.